pkgs/development/python-modules/lintrunner/default.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  maturin,
+  rustPlatform,
+  cargo,
+  rustc,
+  python,
+}:
+
+buildPythonPackage rec {
```

Comment:
I think this tool is an application but not a library.
If so, please use `buildPythonApplication` and move it to `pkgs/by-name/li/lintrunner` instead.

Reply:
It is an application, I've used `buildPythonApplication` instead of `buildPythonPackage`. The typical way to install the application is via `pip`, should the package then be referenced/placed in the python top-level packages? This way it can be installed using `python3Packages.lintrunner` instead of the package name `lintrunner`, or is this bad practice?